src/main/resources/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig/help-useAcceleratedEndpoint.html

```diff
@@ -0,0 +1,4 @@
+<div>This parameter is controling usage of Amazon S3 Transfer Acceleration.
+Amazon S3 Transfer Acceleration enables fast, easy, and secure transfers of files over long distances between your client and an S3 bucket. Transfer Acceleration takes advantage of Amazon CloudFront’s globally distributed edge locations. As the data arrives at an edge location, data is routed to Amazon S3 over an optimized network path.
+<a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/transfer-acceleration.html">https://docs.aws.amazon.com/AmazonS3/latest/dev/transfer-acceleration.html</a>
```

Comment:
The body of the link should be the page title or similar, not a copy of the URL.